recipes-cgl/pacemaker/cluster-glue_%.bbappend

```diff
@@ -0,0 +1,10 @@
+# Copyright (C) 2021, RTE (http://www.rte-france.com)
```

Comment:
There is already a cluster-glue_%.bbappend in recipes-cgl/cluster-glue/cluster-glue_%.bbappend.